### PR TITLE
Add ReqID and job description to member emails <PD-1288>

### PR DESCRIPTION
**Pull request by tdphillips**
Based on the redirect merge branch. If you want to see the actual changes:
https://github.com/DirectEmployers/MyJobs/compare/merge-redirect...pd-1288

### CODE REVIEW

**Comment by aspidites** [comment]
Can you sync this? I assume the DO NOT MERGE can be removed?

**Comment by tdphillips** [comment]
First thing tomorrow.

**Comment by tdphillips** [comment]
Synced.

File: redirect/views.py [hunk 1]
```diff
@@ -275,6 +278,24 @@ def email_redirect(request):
...
 
+    reqid = solr_job.get('reqid', 'None') if solr_job else 'Expired'
+    subject = u'[ReqID: {reqid}] - {subject}'.format(
+        reqid=reqid, subject=subject)
+
+    # We want to ensure both text and html emails get sent, hence what could
+    # be considered a bit of duplication.
+    dashes = "----------------------"
+    description = (solr_job.get('description', job.job_title)
+                   if solr_job
```
Comment (aspidites) [inline redirect/views.py]: These if blocks made me think i was reading Haskell for a minute.

**Comment by aspidites** [comment]
Seems fine to me. Way to test locally or nah?

**Comment by tdphillips** [comment]
I'm improving the test.

**Comment by tdphillips** [comment]
Updated some tests to check for the new additions.

**Pull request merged**